### PR TITLE
Send verification data to backend

### PR DESCRIPTION
**Pull request by vahinim**
Add http post method to send data in business_verification_view.dart
Set geoLocator to get present location.
For now used dummy agent email and store phone, update it while merging into master.

### CODE REVIEW

File: frontend/gpay_agent/lib/business_verification_data/globals.dart [hunk 1]
```diff
@@ -0,0 +1,15 @@
+library gpay_agent.globals;
+import 'package:agent_app/business_verification_data/verification.dart';
+
+final String merchantName = 'Medidoddi Vahini';
+final String address = 'H-no:15-108/1/k, Raghavendra colony, Nagarkurnool, Telangana';
+final String phone = '9381143249';
+
+enum verificationStatus{
```
Comment (trblessy) [inline frontend/gpay_agent/lib/business_verification_data/globals.dart:8]: Why is this enum defined here?

File: frontend/gpay_agent/android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -50,4 +50,6 @@
             android:name="flutterEmbedding"
             android:value="2" />
     </application>
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```
Comment (trblessy) [inline frontend/gpay_agent/android/app/src/main/AndroidManifest.xml:53]: We only need to request one access permission
https://developers.google.com/maps/documentation/android-sdk/location#location_permissions

File: frontend/gpay_agent/ios/Runner/Info.plist [hunk 3]
```diff
@@ -41,5 +41,13 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>NSLocationWhenInUseUsageDescription</key>
+    <string>This app needs access to location when open.</string>
+
+    <key>NSLocationAlwaysUsageDescription</key>
+    <string>This app needs access to location when in the background.</string>
+
+    <key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
```
Comment (trblessy) [inline frontend/gpay_agent/ios/Runner/Info.plist]: Based on above comment you may only need one of these

**Review by trblessy — CHANGES_REQUESTED** [review]

File: frontend/gpay_agent/lib/business_verification_data/globals.dart (hunk 1 above)
Comment (vahinim) [reply frontend/gpay_agent/lib/business_verification_data/globals.dart:8]: I want to access in both verification.dart and business_verification_home_view.dart to send data to back-end. So declared in globals.dart

File: frontend/gpay_agent/android/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (vahinim) [reply frontend/gpay_agent/android/app/src/main/AndroidManifest.xml:53]: Removed ACCESS_COARSE_LOCATION

File: frontend/gpay_agent/ios/Runner/Info.plist (hunk 3 above)
Comment (vahinim) [reply frontend/gpay_agent/ios/Runner/Info.plist]: Removed not required permissions

File: frontend/gpay_agent/lib/business_verification_data/globals.dart (hunk 1 above)
Comment (trblessy) [reply frontend/gpay_agent/lib/business_verification_data/globals.dart:8]: ok

**Review by trblessy — APPROVED** [review]

**Pull request merged**